content/docs/start/experiments/experiment-tracking.md

```diff
@@ -117,7 +117,8 @@ with Live(save_dvc_exp=True) as live:
 </toggle>
 
 After this, each execution of the code will create a <abbr>DVC experiment</abbr>
-containing the results and the changes needed to reproduce it.
+containing the results and the changes needed to reproduce it. Each experiment
+is automatically saved in Git, so you don't need to worry about committing them.
```

Comment:
I still wonder if we should put some notes in places like this with a link to the details :)

Reply:
Like to https://dvc.org/doc/user-guide/experiment-management#how-does-dvc-track-experiments? I think it makes sense, although even adding this sentence gives me pause about starting back towards overloading the content here.

Reply:
yes, also don't have a good solution for this (an admon if we consider this is very important).

Reply:
```suggestion
is [automatically saved in Git](/doc/user-guide/experiment-management#how-does-dvc-track-experiments), so you don't need to worry about committing them.
```

Reply:
Adding a link directly in the text for now. It's not as obvious but doesn't add any space.